src/Uno.UI/Microsoft/UI/Xaml/Controls/RatingControl/RatingControlAutomationPeer.cs

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Uno.UI.Helpers.WinUI;
+using Windows.Foundation;
+using Windows.Globalization.NumberFormatting;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Automation.Provider;
+using RatingControl = Microsoft.UI.Xaml.Controls.RatingControl;
+
+namespace Microsoft.UI.Xaml.Automation.Peers
+{
+	public class RatingControlAutomationPeer : FrameworkElementAutomationPeer, IRangeValueProvider, IValueProvider
+	{
+		public RatingControlAutomationPeer(RatingControl owner) : base(owner)
+		{
+		}
+
+		protected override string GetLocalizedControlTypeCore()
+		{
+			return ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_RatingLocalizedControlType);
+		}
+
+		// Properties.
+		bool IValueProvider.IsReadOnly => GetRatingControl().IsReadOnly;
+
+		string IValueProvider.Value
+		{
+			get
+			{
+				double ratingValue = GetRatingControl().Value;
+				string valueString;
+
+				//string ratingString;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6162540)